CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Improvements
 
+- chore: add api checks with binary compatibility plugin ([#85](https://github.com/getsentry/sentry-kotlin-multiplatform/pull/85))
```

Comment:
CHANGELOG is meant for user-facing changes, no need to put those changes in the changelog.